variables.tf

```diff
@@ -221,6 +247,19 @@ variable "activity_tracker_tags" {
   default     = []
 }
 
+variable "activity_tracker_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module. For more information, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial."
```

Comment:
`to the resources created by the module` -> `to the Activity Tracker instance created by the module`